kroxylicious-api/src/main/java/io/kroxylicious/proxy/config/tls/SslProtocol.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright Kroxylicious Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+package io.kroxylicious.proxy.config.tls;
+
+import java.util.Arrays;
+import java.util.Optional;
+
+public enum SslProtocol {
```

Comment:
We favoured using the word Tls over Ssl in the code, so let's call this TlsProtocol.